### PR TITLE
update guidance

### LINKED ISSUE #3097: adminUser passwordSecret
The install documentation recommends using something like SOPS or similar rather than using the default PASSWORD hash.
The issue is the chart doesn't provide an option to pass along the secret name to use.

So for example create a admin-dashboard-secret this would need to be able to be passed to the HelmRelease in place of adminUser.passwordHash ... so something like adminUser.passwordSecretName ?

### PR DESCRIPTION
**Pull request by sympatheticmoose**
Closes #3097

**What changed?**

**Why was this change made?**

**How was this change implemented?**

**How did you validate the change?**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: website/docs/installation/weave-gitops.mdx [hunk 1]
```diff
@@ -139,6 +139,8 @@ brew install weaveworks/tap/gitops
 
 :::warning
 This command stores a hash of a password.  While this is relatively safe for demo and testing purposes it is recommend that you look at more secure methods of storing secrets (such as [Flux's SOPS integration](https://fluxcd.io/docs/guides/mozilla-sops/)) for production systems.
+
+More guidance and alternative login methods can be found in [Securing access to the dashboard](../../configuration/securing-access-to-the-dashboard).
```
Comment (bigkevmcd) [inline website/docs/installation/weave-gitops.mdx:143]: Should this be a single `..` ?

This is `website/docs/installation/weave-gitops.mdx` and the referenced file is in `website/docs/configration/securing-access-to-the-dashboard.mdx` ?

Comment (bigkevmcd) [reply website/docs/installation/weave-gitops.mdx:143]: I could be misunderstanding how we generate the links when it's deployed tho'

Comment (yiannistri) [reply website/docs/installation/weave-gitops.mdx:143]: Seems to work https://staging.docs.gitops.weave.works/install-update/docs/installation/weave-gitops/#deploying-weave-gitops

**Review by yiannistri — APPROVED** [review]

File: website/docs/installation/weave-gitops.mdx (hunk 1 above)
Comment (sympatheticmoose) [reply website/docs/installation/weave-gitops.mdx:143]: tried single `..` just to check and got `Page Not Found`

**Pull request merged**